python/notebooks/06-memory-and-embeddings.ipynb

```diff
@@ -390,9 +515,126 @@
       "id": "59294dac",
       "metadata": {},
       "source": [
-        "Now you might be wondering what happens if you have so much data that it doesn't fit into your RAM? That's where you want to make use of an external Vector Database made specifically for storing and retrieving embeddings.\n",
+        "Now you might be wondering what happens if you have so much data that it doesn't fit into your RAM? That's where you want to make use of an external Vector Database made specifically for storing and retrieving embeddings. Fortunately, semantic kernel makes this easy thanks to an extensive list of available connectors. In the following section, we will connect to an existing Azure AI Search service that we will use as an external Vector Database to store and retrieve embeddings.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 13,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "from semantic_kernel.connectors.memory.azure_cognitive_search import AzureCognitiveSearchMemoryStore\n",
         "\n",
-        "Stay tuned for that!"
+        "azure_ai_search_api_key, azure_ai_search_url = sk.azure_aisearch_settings_from_dot_env()\n",
+        "kernel.register_memory_store(memory_store=AzureCognitiveSearchMemoryStore(1536,search_endpoint=azure_ai_search_url,admin_key=azure_ai_search_api_key))\n"
```

Comment:
For the `vector_size` = 1536, could we use the named argument, please? It helps clarify for the reader/user if they're not familiar with what "1536" is.

Reply:
My other small request with this line would be to try and create some line wrapping, please? Something like this:

kernel.register_memory_store(
    memory_store=AzureCognitiveSearchMemoryStore(
        vector_size=1536,
        search_endpoint=azure_ai_search_url,
        admin_key=azure_ai_search_api_key
    )
)

Reply:
Hi Evan, I hope this is addressed in  commit 957f3d7d52d05075b1c524f32c5e037713dc73ae